playbooks/templates/.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Differential ShellCheck
+on:  # yamllint disable-line rule:truthy
+  push:
+  pull_request:
+    branches: [ main ]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # required for all workflows
+      security-events: write
+
+    steps:
+      - name: Repository checkout
+        uses: {{ gha_checkout_action }}
+        with:
+          fetch-depth: 0
+
+      - id: ShellCheck
+        name: Differential ShellCheck
+        uses: redhat-plumbers-in-action/differential-shellcheck@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
{%- raw %}
          token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
without the `raw`, jinja will try to process the `{{ secrets.GITHUB_TOKEN }}` and fail